docs/src/guides/micronaut/redis.md

```diff
@@ -0,0 +1,45 @@
+---
+title: "How to Deploy Micronaut on Platform.sh with Redis"
+sidebarTitle: "Micronaut"
+weight: -110
+layout: single
+description: |
+    Configure a Micronaut application with Redis.
+---
+
+To activate Redis and then have it accessed by the Micronaut application already in Platform.sh, it is necessary to modify two files. 
+
+{{< note >}}
+This guide only covers the *addition* of a service configuration to an existing Micronaut project already configured to deploy on Platform.sh. Please see the [deployment guide](/guides/micronaut/deploy/_index.md) for more detailed instructions for setting up app containers and initial projects. 
+{{< /note >}}
+
+## 1. Add the Redis service
+
+In your `.platform/services.yaml` file, include Persistent Redis with a [valid supported version](/configuration/services/redis.md#persistent-redis):
+
+{{< readFile file="src/registry/images/examples/full/redis-persistent.services.yaml" highlight="yaml" >}}
+
+## 2. Add the Redis relationship
+
+In your `.platform.app.yaml` file, use the service name `searchelastic` to grant the application access to Elasticsearch via a relationship:
+
+{{< readFile file="src/registry/images/examples/full/redis-persistent.app.yaml" highlight="yaml" >}}
+
+## 3. Export connection credentials to the environment
+
+Connection credentials for Elasticsearch, like any service, are exposed to the application container through the `PLATFORM_RELATIONSHIPS` environment variable from the deploy hook onward. Since this variable is a base64 encoded JSON object of all of your project's services, you'll likely want a clean way to extract the information specific to Elasticsearch into it's own environment variables that can be easily used by Micronaut. On Platform.sh, custom environment variables can be defined programmatically in a `.environment` file using `jq` to do just that:
+
+```text
+export REDIS_HOST=$(echo $PLATFORM_RELATIONSHIPS | base64 --decode | jq -r ".redisdata[0].host")
```

Comment:
The spacing here is a little more readable that the previous ones.

Reply:
Thank you, fixed